### PR TITLE
Replace Veutify with Vue Prime

### PR DESCRIPTION
**Pull request by ankitvats1**

### CODE REVIEW

File: electron/electron.js [hunk 1]
```diff
@@ -7,8 +7,9 @@ const isDev = process.env.NODE_ENV == "dev"
 function createWindow() {
   // Create the browser window.
   const mainWindow = new BrowserWindow({
-    width: 800,
```
Comment (sanjayaapna) [inline electron/electron.js:10]: For now, it's ok but manage these via env file later.

**Review by sanjayaapna — APPROVED** [review]

File: electron/electron.js (hunk 1 above)
Comment (ankitvats1) [reply electron/electron.js:10]: Sure, I'll fix this in Electron-specific Ticket.

Comment (wildone) [reply electron/electron.js:10]: from memory this can be controlled in package.json

File: src/components/MenuBar.vue [hunk 2]
```diff
@@ -1,24 +1,89 @@
 <template>
```
Comment (wildone) [inline src/components/MenuBar.vue:1]: is this windows controls?

File: src/components/Project/ContentTab/QueryView.vue [hunk 3]
```diff
@@ -1,34 +1,41 @@
 <template>
```
Comment (wildone) [inline src/components/Project/ContentTab/QueryView.vue:1]: You might as well create a folder for views, there will be others. ContentTab/View/Data.vue... etc

**Review by wildone — COMMENTED** [review]
We need to add some tests for behaviours as well. Like collapsing and focus mode. as these do work already.

File: electron/electron.js (hunk 1 above)
Comment (ankitvats1) [reply electron/electron.js:10]: Okay

File: src/components/MenuBar.vue (hunk 2 above)
Comment (ankitvats1) [reply src/components/MenuBar.vue:1]: yes

File: src/components/Project/ContentTab/QueryView.vue (hunk 3 above)
Comment (ankitvats1) [reply src/components/Project/ContentTab/QueryView.vue:1]: Sure

**Pull request merged**